argocd-notifications/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Argo CD Notifications container
+
+FROM quay.io/cybozu/golang:1.17-focal as builder
+
+ENV REPO=argoproj-labs/argocd-notifications
+ENV VERSION=1.2.1
+
+RUN apt-get update && apt-get install ca-certificates
+
+WORKDIR /work
+RUN curl -sSLf "https://github.com/${REPO}/archive/v${VERSION}.tar.gz" | \
```

Comment:
It's better to place `SHELL ["/bin/bash", "-o", "pipefail", "-c"]` when using a pipeline to catch an error at the left-side of the pipe.

Even without it, the right-side "tar" would notice an corrupted input and fail, though.